data/human/jobs.txt

```diff
@@ -3042,3 +3042,131 @@ mission "Bounty Hunting (Marauder X)"
 	on complete
 		payment 750000
 		dialog phrase "generic fleet bounty hunting payment dialog"
+
+
+
+mission "Hunted Warning"
+	invisible
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	to offer
+		"combat rating" > 8103
+		or
+			not "chosen sides"
+			has "main plot completed"
+	on offer
+		log `Got noticed at a spaceport bar as a "famous warrior." Was warned to watch for pirates looking to make a name for themselves or bounty hunters looking to claim a bounty.`
```

Comment:
What do bounty hunters hunt? Bounties. Doesn't seem odd to me for the word "bounty" to be used multiple times. If we were talking about a deer hunter hunting deer, we wouldn't look for different words for deer just for the same of reducing repetition.

Reply:
Fair point. I guess in my mind avoiding the double usage overrides that it isn't really strange considering the circumstances,  which makes sense since I'm sort of a stickler for those types specific writing rules of thumb.